sfaira/data/dataloaders/loaders/d10_1038_s41591_018_0096_5/human_lung_2018_NA_Lambrechts_001.py

```diff
@@ -0,0 +1,35 @@
+import anndata
+import tarfile
+import scipy.io
+import anndata
+import pandas as pd
+
+
+def load(data_dir):
```

Comment:
please use `def load(data_dir, **kwargs):` so that this does not break if it is called with unused arguments